### PR TITLE
start italian translation because why not

### PR DESCRIPTION
**Pull request by adryzz**
🍞

- [x] Translate
- [x] Make sure it's actually translated
- [x] Additional checkbox for additional reward inside my brain

### CODE REVIEW

**Comment by adryzz** [comment]
aight rebased to master. now there are a couple of issues
image
the `à` character isnt drawn correctly (is it present in the font?)

image
The settings menu is too small and stops rendering text

and then some stuff like `rigging` has no translation and i have no clue what to translate it to

**Comment by LunaTheFoxgirl** [comment]
à is not present in Kosugi Maru, it only has ASCII + Japanese font characters. I need to figure out fonts to bundle for both Chinese and extended latin languages.

File: tl/it.po [hunk 1]
```diff
@@ -0,0 +1,1369 @@
...
+msgid "Remove"
+msgstr "Rimuovi"
+
+#: source/creator/panels/parameters.d:307
+msgid "Interpolation Mode"
+msgstr "Modalità Interpolazione"
+
+#: source/creator/panels/parameters.d:308
+# This is kinda of a mess, as in italian you can't say "Nearest <x>", but "Il <x> più vicino"
+msgid "Nearest"
```
Comment (adryzz) [inline tl/it.po:400]: Same thing here as before with the ordering of words

Comment (adryzz) [reply tl/it.po:400]: oh, i'll fix it

**Comment by adryzz** [comment]
Alright, should be all done except for "Rigging"

**Comment by LunaTheFoxgirl** [comment]
That will need to be translated too, somehow.

**Comment by LunaTheFoxgirl** [comment]
Will try to review it later today.

**Pull request merged**